### ISSUE TITLE
Avatars can get broken

### ISSUE THREAD

**Issue by glacials** [glacials/splits-io#623]
Unclear why, but sometimes avatars can get broken then fixed on a logout/login. Could be when changing your avatar on Twitch, the old one goes away immediately.

**Comment by BatedUrGonnaDie** [comment]
Yeah, if you change on twitch I believe the old one returns a 403 or 404.
